test/baselines/reference/eqn.html

```diff
@@ -4,17 +4,15 @@
 <emu-clause id="sec-example">
   <h1 class="first"><span class="secnum">1</span> Header</h1>
   <p>Can refer to eqns inside paragraph text via autolinking: <emu-xref aoid="DateValue" id="_ref_0"><a href="#eqn-DateValue">DateValue</a></emu-xref>.</p>
-  <emu-alg><ol><li>Return <emu-xref aoid="Value" id="_ref_1"><a href="#sec-example">Value</a></emu-xref>(<var>val</var>);
-  </li></ol></emu-alg>
+  <emu-alg><ol><li>Return <emu-xref aoid="Value" id="_ref_1"><a href="#sec-example">Value</a></emu-xref>(<var>val</var>);</li></ol></emu-alg>
 
-  <emu-eqn aoid="Value2"><div>Value2(<var>t</var>)</div><div>      = <emu-xref aoid="DateValue" id="_ref_2"><a href="#eqn-DateValue">DateValue</a></emu-xref>(<var>t</var>) if <emu-xref aoid="Type" id="_ref_3"><a href="https://tc39.es/ecma262/#sec-ecmascript-data-types-and-values">Type</a></emu-xref>(<var>t</var>) is string</div><div>      = <var>t</var></div></emu-eqn>
+  <emu-eqn aoid="Value2"><div>    Value2(<var>t</var>)</div><div>      = <emu-xref aoid="DateValue" id="_ref_2"><a href="#eqn-DateValue">DateValue</a></emu-xref>(<var>t</var>) if <emu-xref aoid="Type" id="_ref_3"><a href="https://tc39.es/ecma262/#sec-ecmascript-data-types-and-values">Type</a></emu-xref>(<var>t</var>) is string</div><div>      = <var>t</var></div></emu-eqn>
```

Comment:
why is this adding spaces?

Reply:
Because they're present [in the input](https://github.com/tc39/ecmarkup/blob/128fe04def0ffe80341d09aba56d9da38f19434a/test/eqn.html#L15).